qemu/tests/blockdev_commit_install.py

```diff
@@ -0,0 +1,55 @@
+import time
+import logging
+import random
+
+from virttest import utils_test
+from virttest import utils_misc
+from virttest.tests import unattended_install
+
+from provider.blockdev_commit_base import BlockDevCommitTest
+
+
+class BlockdevCommitInstall(BlockDevCommitTest):
+    def configure_system_disk(self, tag):
+        pass
+
+
+def run(test, params, env):
+    """
+    Block commit base Test
+
+    1. Install guest
+    2. create 4 snapshots during guest installation
+    3. commit snapshot 3 to base
+    4. installation can be finished after commit
+    """
+    def tag_for_install(vm):
+        tag_for_install_start = "Starting Login Service"
+        try:
+            serial_output = vm.serial_console.get_output()
+            if tag_for_install_start in serial_output:
+                return True
+            else:
+                return False
+        except:
```

Comment:
Better to point out the specific exceptions.

Reply:
have updated.